env/env.ts

```diff
@@ -1,6 +1,6 @@
-export const environment = {
-  production: true,
-  useEmulators: false,
+export default {
+  production: false,
+  useEmulators: true,
```

Comment:
you switched the boolean value. normal ?

Reply:
Locally you should use the emulator and you're not in production. I think this is the right config.

Reply:
I was confused because file was copied from prod file